include/model.h

```diff
@@ -201,6 +184,10 @@ class Op {
   virtual void init(const FFModel&) = 0;
   virtual void forward(const FFModel&) = 0;
   virtual void backward(const FFModel&) = 0;
+  virtual void print_layer(const FFModel& model) = 0;
+  virtual Parameter* get_weight() = 0;
```

Comment:
Instead of call it get_weight(int index), I plan to rename it to get_parameter(int index) since it  weight and bias are all parameters, use weight to call bias may get people confused.

Reply:
This sounds good to me.

Reply:
Done with it.